suite-native/feature-flags/src/useIsUsbDeviceConnectFeatureEnabled.tsx

```diff
@@ -0,0 +1,18 @@
+import { useAtom } from 'jotai';
+
+import { atomWithUnecryptedStorage } from '@suite-native/storage';
+import { isDevelopOrDebugEnv } from '@suite-native/config';
+
+const isDeviceConnectEnabledAtom = atomWithUnecryptedStorage<boolean>(
+    'isDeviceConnectEnabledAtom',
+    isDevelopOrDebugEnv(),
+);
+
+export const useIsUsbDeviceConnectFeatureEnabled = () => {
+    const [isDeviceConnectEnabled, setIsDeviceConnectEnabled] = useAtom(isDeviceConnectEnabledAtom);
```

Comment:
please rename also atom and variables

Reply:
https://github.com/trezor/trezor-suite/pull/9450/commits/1bb307e74205dd3d41e8c503e38d323a75d79217